src/blocked.c

```diff
@@ -684,10 +684,14 @@ static int getBlockedTypeByType(int type) {
 void signalKeyAsReady(redisDb *db, robj *key, int type) {
     readyList *rl;
 
-    /* If no clients are blocked on this type, just return */
+    /* If no clients are blocked on this type, and if no module is blocking
+     * since we don't know which type modules are waiting for, just return. */
```

Comment:
```suggestion
    /* The following two situations will quickly return:
     * 1, The type would never be blocked.
     * 2, No clients are blocked on this type. (Notice: because all Module
     * block types are BLOCKED_MODULE, so it needs to be judged every time
     * to handle the case which user block key in MODULE, such as blockonkeys.popall
     * but wake up in NORMAL commands, such as BLPOP */
```

Reply:
Thanks for the suggestion. The module which defines `blockonkeys.popall` part of the tests, so I don't think it's appropriate to refer to it here. I can rewrite it a bit.

Reply:
@zuiderkwast ok, please update it